guide/additional-info/changes-in-v14.md

```diff
@@ -6,27 +6,27 @@ v14 requires Node 16.9 or higher to use, so make sure you're up to date. To chec
 
 ### Builders are now included in v14
 
-If you previously had `@discordjs/builders` manually installed it's _highly_ recommended that you uninstall the package to avoid package naming conflicts.
+If you previously had `@discordjs/builders` or `@discordjs/rest` manually installed, it's _highly_ recommended that you uninstall the packages to avoid package naming conflicts.
```

Comment:
```suggestion
If you previously had `@discordjs/builders` or `@discordjs/rest` manually installed, it's _highly_ recommended that you uninstall the packages to avoid package version conflicts.
```